pkg/controller/restore_controller.go

```diff
@@ -58,18 +58,15 @@ var nonRestorableResources = []string{
 
 	// Don't ever restore backups - if appropriate, they'll be synced in from object storage.
 	// https://github.com/heptio/velero/issues/622
-	"backups.ark.heptio.com",
```

Comment:
@nrb @carlisia I think removing these are fine, since users should've migrated their CRDs, and we won't be processing the *.heptio.com API group anymore - make sense?

Reply:
I agree users should have, though I'm unsure that everyone will.

That said, since our controllers won't actually process anything in the old API group, the restored Ark backups will be inert, so it's not going to trigger a cascade.

Reply:
Yep, that was my thinking.